zedd-app/src/components/AppBody.tsx

```diff
@@ -49,6 +55,88 @@ const useStyles = makeStyles((theme) => ({
   },
 }))
 
+export function DateRangePicker(state: AppState) {
```

Comment:
This component only uses state.showing. To make it reusable it should take two properties, `value: Interval` and `onChange`.

As it would then be independent of the AppBody component, we can extract it into it's own file.